### DISCUSSION TITLE
请问"定时开关“怎么理解？好像无论是否打开这个开关都会执行

### DISCUSSION THREAD

**Discussion by awayme** [qd-today/qd#490] [General]
任务里的定时设置，“定时开关”不管是否选择，任务都会在“下次运行”时候执行。不太理解这个开关控制的是什么？
翻了了所有的文档/手册/讨论和Issue都没找到解释，看代码有没能力，有没有懂阿？

**Comment by a76yyyy** [comment]
- 关闭定时开关，框架会按照默认的24小时间隔策略**定期**执行。
- 开启定时开关，用户可以选择让任务在某个时间点**定点**执行，或者按照cron设置**定点**执行

**Reply by awayme** [reply]
明白了，谢谢解释
